pyproject.toml

```diff
@@ -12,12 +12,12 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.10"
-dcicutils = "^6.8.0"
+dcicutils = "^6.9.9.0b2"
 click = "^7.1.2"
 cron-descriptor = "^1.2.31"
 cryptography = "39.0.0" # Required for AWS Cognito JWT decode (PyJWKClient)
 pyDes = "^2.0.1"
-PyJWT = "2.5.0"
+PyJWT = "^2.5.0"
 Jinja2 = "2.10.1"
 MarkupSafe = "1.1.1"
```

Comment:
```suggestion
# TODO: I think this can update to ">=1.1.1,<3" so that it can move to version 2.x.
#       Doing so will mean adjusting the Jinja2 constraint as well. -kmp 23-Feb-2023
MarkupSafe = "1.1.1"
```